seqr/utils/elasticsearch/constants.py

```diff
@@ -172,6 +174,17 @@
     }
 }
 
+
+def _get_phenotype_priority_ranks_by_gene(families, *args):
+    from seqr.utils.elasticsearch.utils import InvalidSearchException
+    if len(families) > 1:
+        raise InvalidSearchException('Sorting on multiple families search results is not supported.')
```

Comment:
We need a clearer error message: `Phenotype sort is only supported for single-family search`